docs/release_docs/Pr-ci-cd_ReleaseFlow.md

```diff
@@ -0,0 +1,26 @@
+## PR-CI-CD
+ 
+ PR-CI-CD can be trigger by creating a pull request from ``` Feature/* ``` branch to merge into develop branch.
+ ![PR-ci-cd_workflow](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/pr-ci-cd_workflow.png?raw=true)
+ ![pr](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/Pr-ci-cd_example.png?raw=true)
+ 
+### JOBS
+
+ These are the jobs used in PR-ci-cd workflow
+ 
+ #### CodeScan-ESLint
+  In this job we are analyzes the code developer wrote against some rules for stylistic or programmatic errors.
+ 
+  * This job use static code analysis tool which identify problematic patterns found in application source code.
```

Comment:
This job uses a static code analysis tool which identifies problematic patterns found in application source code.